subworkflows/nf-core/fusion_star_arriba.nf

```diff
@@ -0,0 +1,50 @@
+/*
+ * Fusion calling with STAR and Arriba
+ */
```

Comment:
I think this should be renamed as arriba.nf

Reply:
done!